package/yast2-country.changes

```diff
@@ -1,3 +1,10 @@
+-------------------------------------------------------------------
+Mon Dec  4 11:30:50 UTC 2017 - jsrain@suse.cz
+
+- do not touch RC_* variables in sysconfig/language (they still
+  keep in use, limiting the meaning to default shell (fate#319454)
```

Comment:
This is hard to understand. What is the impact for the user/admin? Can we name all the vars (RC_LANG only?) instead of using a wild card?